sdks/python/container/license_scripts/dep_urls_py.yaml

```diff
@@ -77,8 +77,6 @@ pip_dependencies:
     license: "https://raw.githubusercontent.com/mtth/hdfs/master/LICENSE"
   httplib2:
     license: "https://raw.githubusercontent.com/httplib2/httplib2/master/LICENSE"
-  mmh3:
```

Comment:
Actually should we keep this since it will be in the container?

Reply:
oh whoops, will add that back in